src/collections/utils.zig

```diff
@@ -5,12 +5,16 @@ const std = @import("std");
 /// Since there's nothing we can do to recover from such an issue,
 /// it's best to always exit the process with a nice message than to
 /// propagate unrecoverable errors all over the compiler.
-pub fn exitOnOom() noreturn {
-    const oom_message =
-        \\I ran out of memory! I can't do anything to recover, so I'm exiting.
-        \\Try reducing memory usage on your machine and then running again.
-    ;
+pub fn exitOnOom(err: std.mem.Allocator.Error) noreturn {
+    switch (err) {
+        error.OutOfMemory => {
```

Comment:
Oh, this is cool. So this makes sure it never accidentally swallows other kinds of errors.